### PR TITLE
Feature game

### PR DESCRIPTION
**Pull request by CornetS28**
# Game feature with player-player feature
Dear TSE,
I've listed down the features we've implemented
- Players can enter their names to select the signs. i.e 'X' and 'O'
- Game reset feature
- Used module system using IIFEs instead of the ES6 modules as they are introduced later in the section/course.
- IIFE includes -> domManager, gameBoard, game
- Factory functions -> Player

TODOs [We'll implement these features once we've finished the minimum requirements]
- [ ] Game with player-AI feature. (Already implemented but on a separate branch)
- [ ] Styling - We realise that the projects should look good and that's why we've kept it for the end so that we can finish it without any pressure.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <div class="game">
+      <!-- <button id="btn-1">1</button>
+      <button id="btn-2">2</button>
+      <button id="btn-3">3</button>
+      <button id="btn-4">4</button>
+      <button id="btn-5">5</button>
+      <button id="btn-6">6</button>
+      <button id="btn-7">7</button>
+      <button id="btn-8">8</button>
+      <button id="btn-9">9</button> -->
```
Comment (JamesOkunlade) [inline index.html]: Don't leave unused codes as comments in your codebase.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  const gameView = document.querySelector('.game');
+  const board = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+
+  return {
+    startGame: () => {
+      player1View.innerHTML = '';
+      player2View.innerHTML = '';
+
+      const playX = prompt('Enter a name for X'); // eslint-disable-line no-alert
+      const playY = prompt('Enter a name for O'); // eslint-disable-line no-alert
```
Comment (JamesOkunlade) [inline main.js]: Do all dom tasks in a separate file. Don't mix dom manipulation with game logic.

File: index.html (hunk 1 above)
Comment (KaushikShivam) [reply index.html]: Thank you, James. It was careless of us to miss it. We've fixed it. 👍

File: main.js (hunk 2 above)
Comment (KaushikShivam) [reply main.js]: Thanks. We've made the changes

**Comment by KaushikShivam** [comment]
> **Status: Approved**
> Hello @CornetS28 @KaushikShivam ✋
> 
> Nice work so far 👏 , its a 👍 from me as you implemented changes the previous reviewer requested.
> 
> Note: Please try to update your UI little more as it is not square at this moment and it does not looks good that way.
> 
> Happy Coding
> Adnan Afsari

Dear Adnan,

We've made the necessary changes except the need to make separate files for the modules as they as taught later in the section. We've have implemented modules using the IIFE pattern as we are
asked. Please refer to the updated pull request description. :)

And yes, we'll make the UI super awesome. We will just do it on weekends instead of the normal coding time. We'll be happy to share it once it's complete. Thanks :D

**Pull request merged**